src/platform/elements/form/FieldInteractionApi.ts

```diff
@@ -311,7 +318,8 @@ export class FieldInteractionApi {
     public setTooltip(key: string, tooltip: string): void {
         let control = this.getControl(key);
         if (control) {
-            control.tooltip = tooltip;
+            control.setTooltip(tooltip);
+            this.form.novoConfigChange.emit({tooltip: tooltip});
```

Comment:
Make a private method to do this, we will need some defensive programming around this.

```
private triggerEvent(event: IFieldInteractionEvent) {
  if (this.form && this.form.fieldInteractionEvents) {
    this.form.fieldInteractionEvents.emit(event);
  }
}
```

Rename the `novoConfigChange` to `fieldInteractionEvents` to be a bit more generic. Then the data model should be like:

```
{
   controlKey: string;
   prop: string;
   value: any;
}
```

then in  each method (not just the ones you need) after the FI do the `this.triggerEvent({ controlKey: key, prop: 'tooltip', value: 'New Tooltip' });`